sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/EndpointAccessResource.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> The endpoint access for the target resource. </summary>
+    public partial class EndpointAccessResource
```

Comment:
This model is not really a resource - therefore could we drop the suffix from this model?

Reply:
good question, to this kind of non-resource model that swagger defined, what can we do? Should we have a generic rule to remove the suffix? Here the original name 'EndpointAccessResource' is not good, but remove the suffix also not a good name...

Reply:
Since the `resource` suffix change is ongoing, I'd like to level it as is now